scripts/lib.sh

```diff
@@ -170,7 +170,7 @@ create_summary_file()
 	local -r filename="osbuilder.yaml"
 	local file="${dir}/${filename}"
 
-	local -r now=$(date '+%Y-%m-%dT%T.%N%zZ')
+	local -r now=$(date -u -d@${SOURCE_DATE_EPOCH:-$(date +%s)} '+%Y-%m-%dT%T.%N%zZ')
```

Comment:
maybe I'm missing something, but I run this command in my system and I get:

```
$ date -u -d@${SOURCE_DATE_EPOCH:-$(date +%s)} '+%Y-%m-%dT%T.%N%zZ'
2019-08-08T14:20:06.000000000+0000Z
```

`000000000+0000Z` is that correct?

Reply:
+0000Z is the UTC timezone. That is correct.
.000000000 were the subseconds. I now added a .%N to not round them away.

Reply:
Now it looks better, thanks